src/components/form/formComponent.jsx

```diff
@@ -22,6 +23,7 @@ export function createForm(config) {
             submitLabel: 'send',
             showLabels: false,
             skipError: false,
+            fieldValidationDebounce: 1000,
```

Comment:
This is not configurable? 😛

Reply:
Not yet. Would have to be configurable via config the same way the password policy is. It's "configurable" as far as it's a Prop of the component though.